packages/lib/src/components/FeedbackWidget/FeedbackWidget.tsx

```diff
@@ -242,7 +242,7 @@ export function FeedbackWidget({
               return (
                 <Stack key={key}>
                   <ActionIcon
-                    color="blue"
+                    color={theme.colors.blue[6]}
```

Comment:
I'm a bit confused, this is still fixed to a blue tone right? 🤔

Reply:
Changed as discussed to theme.primary :)